serverless.yml

```diff
@@ -149,26 +149,46 @@ custom:
       test: 'Jobs-test'
       prod: 'Jobs'
   customActionsURLs:
-    GetMediaInfo:
-      local: '${file(env.yml):local.OFFLINE_API_BASE_URL}api/media-info'
+    ChangeRefreshTokenPeriod:
+      local: '${file(env.yml):local.OFFLINE_API_BASE_URL}api/security/change-refresh-token-period'
       dev: !Sub
         - 'https://${gatewayId}.execute-api.${region}.${suffix}/${opt:stage, self:provider.stage}/${path}'
         - gatewayId: !Ref ApiGatewayRestApi
           region: !Ref AWS::Region
           suffix: !Ref AWS::URLSuffix
-          path: 'api/media-info'
+          path: 'api/security/change-refresh-token-period'
       test: !Sub
         - 'https://${gatewayId}.execute-api.${region}.${suffix}/${opt:stage, self:provider.stage}/${path}'
         - gatewayId: !Ref ApiGatewayRestApi
           region: !Ref AWS::Region
           suffix: !Ref AWS::URLSuffix
-          path: 'api/media-info'
+          path: 'api/security/change-refresh-token-period'
       prod: !Sub
         - 'https://${gatewayId}.execute-api.${region}.${suffix}/${opt:stage, self:provider.stage}/${path}'
         - gatewayId: !Ref ApiGatewayRestApi
           region: !Ref AWS::Region
           suffix: !Ref AWS::URLSuffix
-          path: 'api/media-info'
+          path: 'api/security/change-refresh-token-period'
+    ExampleOfUseAuthorizer:
+      local: '${file(env.yml):local.OFFLINE_API_BASE_URL}api/examples/use-authorizer'
+      dev: !Sub
+        - 'https://${gatewayId}.execute-api.${region}.${suffix}/${opt:stage, self:provider.stage}/${path}'
+        - gatewayId: !Ref ApiGatewayRestApi
+          region: !Ref AWS::Region
+          suffix: !Ref AWS::URLSuffix
+          path: 'api/examples/use-authorizer'
+      test: !Sub
+        - 'https://${gatewayId}.execute-api.${region}.${suffix}/${opt:stage, self:provider.stage}/${path}'
+        - gatewayId: !Ref ApiGatewayRestApi
+          region: !Ref AWS::Region
+          suffix: !Ref AWS::URLSuffix
+          path: 'api/examples/use-authorizer'
+      prod: !Sub
+        - 'https://${gatewayId}.execute-api.${region}.${suffix}/${opt:stage, self:provider.stage}/${path}'
+        - gatewayId: !Ref ApiGatewayRestApi
+          region: !Ref AWS::Region
+          suffix: !Ref AWS::URLSuffix
+          path: 'api/examples/use-authorizer'

```

Comment:
Try to find a way to unite these custom variables because they have the same structure. If there is no way to do it, keep it as is.